src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -316,17 +317,30 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			if (selection && !selection.isEmpty()) {
 				const textModel = editorControl?.getModel() as TextModel;
 				const value = textModel?.getValueInRange(selection);
-				return value || '';
+				let linkMatches = value.match(linksRegex);
```

Comment:
Do we need another ?. operator on **value** here?

Reply:
I do not think we do. The selection cannot be empty, however, the value.match can return null if not it does not find the matching pair which in the cases it doesn't would either return the value that the user is selected (which is not a link) or an empty string which makes sense.

Reply:
Well, but textModel could (supposedly) be undefined.

Since the code is taking that case into account we should make sure we handle that correctly. Right now if that (or editorControl) was undefined then we'd get a null ref error on `value.match(...)` since value would be undefined.

Reply:
Ah, that would make sense. I will be sure to make it handle correctly then.